README.md

```diff
@@ -84,10 +84,10 @@ to connect.
 Black Magic Debug App can talk to
 
 * Black Magic Probe firmware probes via the USB-serial port
-* ST-LinkV2 and V3 with recent firmware
+* ST-Link v2, v2.1 and v3 with recent firmware
 * CMSIS-DAP compatible probes
-* JLINK probes
-* FTDI MPSSE based probe.
+* J-Link probes
+* FTDI MPSSE based probes
 
 When connected to a single BMP supported probe, starting "blackmagic" w/o any
```

Comment:
Some additional feedback we've had suggested in private, is to switch the quotes over to backticks for the binary name

Reply:
Thats a good sugestion, done.